src/components/dashboard/HoloFuelCard.vue

```diff
@@ -68,29 +39,70 @@ const items = computed((): DashboardCardItem[] => [
     label: t('holofuel.redeemable'),
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     value:
-      !isErrorPredicate(props.data) && Number(props.data.available) && Number(props.data.redeemable)
+      !isErrorPredicate(props.data) && Number(props.data.redeemable)
         ? formatCurrency(Number(props.data.redeemable))
         : 0,
-    isActive: false
+    isActive: true
```

Comment:
```suggestion
    isActive: props.data.redeemable
```

Should the `isActive` be dependant on whether the there is a redeemable amount?

Reply:
No, this is for the label and value, not for the button. Those should be always enabled now, but... I will remove the whole property as this was needed only as a temporary solution until the feature is implemented. Thanks